python/ray/autoscaler/autoscaler.py

```diff
@@ -53,8 +53,9 @@
     # Cloud-provider specific configuration.
     "provider": ({
         "type": (str, REQUIRED),  # e.g. aws
-        "region": (str, REQUIRED),  # e.g. us-east-1
-        "availability_zone": (str, REQUIRED),  # e.g. us-east-1a
+        "region": (str, OPTIONAL),  # e.g. us-east-1
```

Comment:
I guess the OPTIONAL and REQUIRED fields are going to depend on the backend, so as we add more backends we'll have to just make everything optional, is that right @ericl?

Reply:
Yeah, optional sounds good.